app/avo/resources/rubygem_resource.rb

```diff
@@ -6,14 +6,22 @@ class RubygemResource < Avo::BaseResource
   }
 
   self.find_record_method = lambda { |model_class:, id:, params:| # rubocop:disable Lint/UnusedBlockArgument
-    model_class.find_by!(name: id)
+    # In case of perfoming action `id` becomes an array of `ids`
```

Comment:
this feels wrong. can you raise an issue in the avo repo? If raw IDs are being passed in, avo should be calling `find` with those itself, rather than delegating to our `find_record_method` which imo is singular as the name implies